scripts/test.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2019 Advanced Micro Devices, Inc. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''
+GFX reconstruct build script
+'''
+
+import argparse
+import copy
+import distutils.version
+import importlib
+import os
+import platform
+import re
+import shutil
+import subprocess
+import sys
+
+
+def is_windows():
+    '''
+    Check if the system is Windows
+    '''
+    return 'windows' == platform.system().lower()
+
+
+BUILD_ROOT = os.path.abspath(os.path.join(
+    os.path.split(os.path.abspath(__file__))[0], '..'))
+ALL_TESTS = {
```

Comment:
Please document here what the value field represents.  It wasn't clear until I'd gotten all the way through the code.

Also, since `run_test()` accepts a value of `None` to indicate no test arguments, why don't each of these have a value of `None`?

Reply:
These are the tests to run when no test exe is passed to the script, why do we need to add None here?